DESCRIPTION

```diff
@@ -23,7 +23,8 @@ Depends:
 Imports: 
   stats (>= 3.6),
   hypergeo (>= 1.2-13),
-  purrr (>= 0.3.3)
+  purrr (>= 0.3.3),
+  coin (>= 1.3-1)
```

Comment:
Ik ben benieuwd, wat is coin voor package? ik kan er geen documentatie van vinden op cran, zoek ik verkeerd?

Reply:
https://cran.r-project.org/web/packages/coin/index.html. Het meest belangrijke is https://www.rdocumentation.org/packages/coin/versions/1.3-1/topics/SurvivalTests